src/ansi-c/literals/convert_character_literal.cpp

```diff
@@ -53,7 +54,7 @@ exprt convert_character_literal(
       for(unsigned i=0; i<value.size(); i++)
       {
         mp_integer z=(unsigned char)(value[i]);
-        z=z<<((value.size()-i-1)*8);
+        z = z << ((value.size() - i - 1) * CHAR_BIT);
```

Comment:
Why CBMC's compiled char size here, not the char size we're working with?

Reply:
Good point - I was thinking this was what CBMC read a text string (which is true), but of course that text string is written for the platform that we are analyzing for.